### ISSUE TITLE
Evaluate some libraries

### ISSUE THREAD

**Issue by svenstaro** [svenstaro/rust-web-boilerplate#2]
- [ ] https://github.com/kriomant/mockers
- [ ] https://github.com/fuchsnj/validate
- [ ] https://github.com/rustless/valico
- [ ] https://github.com/Keats/validator
- [ ] https://github.com/ChrisBuchholz/accord

**Comment by pythoneer** [comment]
could you elaborate why to rule out (validate, valico)? I guess valico does not really fit to the rocked "way" of doing things, right? I used lettre in one of my projects and it was not to bad and did all i needed it to do.

**Comment by svenstaro** [comment]
Yeah, valico doesn't fit and validate seems too underdeveloped and unmaintained. I want something that will be fixed upstream if it breaks.
